packages/hardhat-core/package.json

```diff
@@ -68,7 +68,7 @@
     "@types/glob": "^7.1.1",
     "@types/keccak": "^3.0.1",
     "@types/lodash": "^4.14.123",
-    "@types/mocha": "^9.1.0",
+    "@types/mocha": "^10.0.0",
```

Comment:
Maybe `>=9.1.0` so that we don't have to release a new major?  I think that for this one in particular (and maybe `@types/` in general), we can be flexible

Reply:
Done.